### ISSUE TITLE
Automating new bugfix releases

### ISSUE THREAD

**Issue by yebai** [TuringLang/Turing.jl#1631]
We currently manually make new releases for Turing libraries. Maybe this can be automated via Github Actions? For example, if the following conditions are met, we trigger a new release automatically: 

1. version of the package is increased by a PR (in practice, we can limit to bugfix releases for automated releases only).  
2. PR is merged by `bors r+`

**Comment by rikhuijzer** [comment]
> version of the package is increased by a PR

Yes, this is something that I was thinking about too. Would be very nice
